easel/lib/easel_provider.dart

```diff
@@ -814,12 +813,27 @@ class EaselProvider extends ChangeNotifier {
       uploadThumbnailResponse = uploadResponse.getOrElse(() => StorageResponseModel.initial());
     }
 
-    final response = await repository.uploadFile(
-      file: _file!,
-      onUploadProgressCallback: (value) {
-        _uploadProgressController.sink.add(value);
-      },
-    );
+    final whereToUpload = QuickNode.listOfQuickNodeAllowedExtension().contains(fileExtension.toLowerCase());
```

Comment:
The name doesn't justifies what it is doing. Maybe you should use `shouldUploadToQuickNode`

Reply:
done

Reply:
done